fe/fe-core/src/main/java/org/apache/doris/datasource/EsExternalCatalog.java

```diff
@@ -125,26 +122,6 @@ protected void initLocalObjectsImpl() {
         }
     }
 
-    @Override
```

Comment:
Why remove this?

Reply:
> Why remove this?

I found that the logic of EsExternalCatalog is basically the same as that of the parent class, except that the data source is `default_db`. So I directly used the method of the parent class and it works.